README.md

```diff
@@ -64,6 +64,14 @@ bm.hasBuildCompleted().waitFor();
 #### Image
 Wrapper class for url specified images. Its purpose is to parse them or turn them into ImageStream objects.
 
+##### Specifying Maven
+In some images Maven needs to be activated, for example on RHEL7 via script `/opt/rh/rh-maven35/enable`. 
+This can be controlled by properties.
+
+ * `xtf.maven.script` - path to Maven activation script. Defaults to `/opt/rh/rh-maven35/enable` if not set.
```

Comment:
I dislike the name, what about `xtf.maven.activation-script` or `xtf.maven.activation_script`?

Reply:
I've checked other configuration attributes and underscore is more commonly used on this project. Let's go with `xtf.maven.activation_script`